controller/controller_test.go

```diff
@@ -897,20 +895,7 @@ func TestControllerMutation(t *testing.T) {
 					panic("bad wanted IP in loadbalancer status")
 				}
 			}
-		}
-
-		if t.Failed() {
-			// Don't run more test cases if we've already failed, to
-			// keep the output readable.
-			break
-		}
-
-		// Shuffle the input vector, and run again.
-		for x := range tests {
```

Comment:
why this part was removed? I don't think it's part of this change and I guess we still want it.
If a commit says "move to subtests", it should do that and nothing more.

Reply:
Right, out of the scope of this PR. Will add it back.